yt/yt/ytlib/query_tracker_client/records/query.yaml

```diff
@@ -219,14 +219,20 @@ types:
         column_type: Any
 
 
-  - type_name: TFinishedQueryByStartTime
+  - type_name: TFinishedQueryByAcoAndStartTime
     fields:
       # Key.
 
-      - cpp_name: StartTime
-        cpp_type: TInstant
-        column_name: start_time
-        column_type: Timestamp
+      - cpp_name: AccessControlObject
+        cpp_type: TString
+        column_name: access_control_object
+        column_type: String
+        sort_order: Ascending
+
+      - cpp_name: MinusStartTime
+        cpp_type: i64
+        column_name: minus_start_time
+        column_type: Int64
         sort_order: Ascending
```

Comment:
I wasn't able to find a reason on why is it `minus_start_time` and not `start_time`?
Can't we do `start_time` with proper sort_order and code adjustments?

Reply:
We need to retrieve the last N queries from our index tables. Currently, dyntables can only do this by performing a full scan. If we need only the first N queries, they can perform this task much more efficiently.